int32 id1
int32 id2
time stamp1
time stamp2
--- 
